assets/src/edit-story/components/panels/alignment/alignment.js

```diff
@@ -204,7 +219,7 @@ function ElementAlignmentPanel({ selectedElements, pushUpdate }) {
   );
 
   return (
-    <ElementRow ref={ref}>
+    <ElementRow ref={ref} aria-label={__('Alignment Panel', 'web-stories')}>
```

Comment:
Hm, why does the row have a label?

Reply:
Just want to check the panel is rendered 😄

Reply:
But it doesn't make sense to have a label on the row - labels are for accessibility only.

If you need to check if the panel exists, look for something that must always be inside the panel, just as you would if you were visually checking that it exists. E.g. the panel title or some button.

Reply:
Makes sense for me.